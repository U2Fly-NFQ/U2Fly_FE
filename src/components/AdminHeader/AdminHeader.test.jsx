import renderer from 'react-test-renderer'
import React from 'react'
import AddminHeader from './'

describe('Test snapshot', () => {
  it('Should match snapshot', () => {
    const tree = renderer.create(<AddminHeader />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
