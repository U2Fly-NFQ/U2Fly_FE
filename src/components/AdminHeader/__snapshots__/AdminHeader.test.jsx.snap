// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot Should match snapshot 1`] = `
<header
  className="ant-layout-header admin-header"
>
  <div
    className="ant-row admin-header-content"
    style={Object {}}
  >
    <div
      className="ant-col admin-header-content-left ant-col-xs-0 ant-col-sm-12"
      style={Object {}}
    >
      <span
        className="ant-input-group-wrapper ant-input-search ant-input-search-with-button"
      >
        <span
          className="ant-input-wrapper ant-input-group"
        >
          <input
            className="ant-input"
            hidden={null}
            onBlur={[Function]}
            onChange={[Function]}
            onCompositionEnd={[Function]}
            onCompositionStart={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Search in app"
            style={null}
            type="text"
            value=""
          />
          <span
            className="ant-input-group-addon"
          >
            <button
              className="ant-btn ant-btn-primary ant-input-search-button"
              disabled={false}
              onClick={[Function]}
              onMouseDown={[Function]}
              type="button"
            >
              <span>
                Search
              </span>
            </button>
          </span>
        </span>
      </span>
    </div>
    <div
      className="ant-col admin-header-content-right ant-col-xs-24 ant-col-sm-12"
      style={Object {}}
    >
      <span
        className="ant-avatar ant-avatar-lg ant-avatar-circle ant-avatar-icon ant-dropdown-trigger profileMenu"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={Object {}}
      >
        <span
          aria-label="user"
          className="anticon anticon-user"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="user"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
            />
          </svg>
        </span>
      </span>
    </div>
  </div>
</header>
`;
